### ISSUE TITLE
Bug when passing through rotations

### ISSUE THREAD

**Issue by BigBastard** [msummers123/Movecraft-3#172]
I've got a video demonstrating this:

https://www.youtube.com/watch?v=hOn9rJdTnKg

**Comment by BigBastard** [comment]
Really you're not even going to look through it?

**Comment by TylerS1066** [comment]
BaccaYarro has alot of things to do, and around the holidays I'm sure he's busy.  That along with the fact that he will be gone for January mustn't be easy on him.

There is also no guarantee that Movecraft will work perfect, and while it does a majority of the time it is prone to having bugs.

**Comment by msummers123** [comment]
Yes, sorry for the late reply I have unfortunately been rather busy. What mod is it you are using? It's not movecraft, which is a plugin not a mod. However I am very curious what it is. It seems to allow redstone function while allowing smooth motion of the craft, which I did not think any mod was capable of. I would love to hear more about this mod.
